### ISSUE TITLE
sendPoll is not working

### ISSUE THREAD

**Issue by manteguinha** [open-wa/wa-automate-nodejs#3045]
### Are you using the latest version of the library?

- [X] I have checked and am using the latest version of the library.

### What type of session are you experiencing this issue on?

Multi-device and I have set multiDevice to true in my config OR am using the --multi-device flag

### What type of host account are you experiencing this issue on?

Business account

### Mode

My own code

### Current Behavior

Error: Evaluation failed: TypeError: Cannot read properties of undefined (reading 'id')
 at b (https://web.whatsapp.com/lazy_loaded_low_priority_components.c4de6b8d051267e9fef1.js:2:44797)
at https://web.whatsapp.com/lazy_loaded_low_priority_components.c4de6b8d051267e9fef1.js:2:44528
at Generator.next (<anonymous>)
at t (https://web.whatsapp.com/vendor1~app.0c8e960e5dfd7020f562.js:2:66483)
at s (https://web.whatsapp.com/vendor1~app.0c8e960e5dfd7020f562.js:2:66694)
at https://web.whatsapp.com/vendor1~app.0c8e960e5dfd7020f562.js:2:66753
at Y (https://web.whatsapp.com/app.af1c9e40459f98eb9cf6.js:5:140402)
at new y (https://web.whatsapp.com/app.af1c9e40459f98eb9cf6.js:5:132969)
at https://web.whatsapp.com/vendor1~app.0c8e960e5dfd7020f562.js:2:66634
at y (https://web.whatsapp.com/lazy_loaded_low_priority_components.c4de6b8d051267e9fef1.js:2:44712)
at ExecutionContext._ExecutionContext_evaluate (/home/mantega/bots/MantegaBot/node_modules/puppeteer-core/lib/cjs/puppeteer/common/ExecutionContext.js:247:15)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
at async ExecutionContext.evaluate (/home/mantega/bots/MantegaBot/node_modules/puppeteer-core/lib/cjs/puppeteer/common/ExecutionContext.js:125:16)

### Steps To Reproduce

`await.client.sendPoll(chatId, "Title", ["Test", "test"])`

### create() code OR full CLI command + CONFIG

```markdown
sessionId: 'session',
        headless,
        multiDevice: true, 
        qrTimeout: 0,
        authTimeout: 0,
        hostNotificationLang: 'pt-br',
        skipUpdateCheck: true,
        cachedPatch: true,
        cacheEnabled: false,
        restartOnCrash: start,
        useChrome: true,
        stickerServerEndpoint: true,
        killProcessOnBrowserClose: true,
        throwErrorOnTosBlock: false,
        chromiumArgs: [
            '--no-sandbox',
            '--disable-setuid-sandbox',
            '--aggressive-cache-discard',
            '--disable-cache',
            '--disable-gl-drawing-for-tests',
            '--disable-application-cache',
            '--disable-offline-load-stale-cache',
            '--disk-cache-size=0'
        ]
```

### DEBUG INFO

```markdown
┌─────────────────────┬────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│       (index)       │                                                                   Values                                                                   │
├─────────────────────┼────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│     WA_VERSION      │                                                                 '2.2306.7'                                                                 │
│       PAGE_UA       │ 'WhatsApp/2.2147.16 Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/104.0.0.0 Safari/537.36' │
│ WA_AUTOMATE_VERSION │                                                                  '4.55.2'                                                                  │
│   BROWSER_VERSION   │                                                       'HeadlessChrome/110.0.5481.77'                                                       │
│         OS          │                                                                'Linux 5.15'                                                                │
│      START_TS       │                                                               1676042312047                                                                │
│      RAM_INFO       │                                                      'Total: 1.87 GB | Free: 1.17 GB'                                                      │
│    PPTR_VERSION     │                                                                  '19.6.3'                                                                  │
└─────────────────────┴────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
```

### Environment

```markdown
- OS: Ubuntu 22.04.1 LTS
- Node: v18.14.0
- npm: 9.3.1
```

### Anything else?

![image](https://user-images.githubusercontent.com/63943591/218129998-e12bc5aa-a0d1-4115-9ef1-3f03ed586f69.png)

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
try {
	execSync(cmd, {stdio: 'inherit'})
}  catch (error) {
    console.log(`Status Code: ${error.status} with '${error.message}'`);
 }
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN && npm run types:publish `);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.55.2](https://github.com/open-wa/wa-automate-nodejs/compare/4.55.1...4.55.2) (2023-01-21)

- 📝 docs update [`6fbd9a3`](https://github.com/open-wa/wa-automate-nodejs/commit/6fbd9a37a5311a3244195f5192c5a5204042a4d2)
- Release 4.55.2 [`df8086b`](https://github.com/open-wa/wa-automate-nodejs/commit/df8086b363db18a2b93dc7b100878ed604d9d2a9)
- 🔧 added `qrUrl` to list of default ev filter #3016 [`7d4911a`](https://github.com/open-wa/wa-automate-nodejs/commit/7d4911a24e20c0451427f942844cf6bfbe37f648)
- updated types-only package [`da397f2`](https://github.com/open-wa/wa-automate-nodejs/commit/da397f20cf659855c246bbf5039a3076cbec63b5)

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
try {
	execSync(cmd, {stdio: 'inherit'})
}  catch (error) {
    console.log(`Status Code: ${error.status} with '${error.message}'`);
 }
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN && npm run types:publish `);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.55.4](https://github.com/open-wa/wa-automate-nodejs/compare/4.55.2...4.55.4) (2023-02-10)

- Release 4.55.4 [`6dcf17e`](https://github.com/open-wa/wa-automate-nodejs/commit/6dcf17e8bc3f2b6d27b44b4c157bf965c9e8e947)
- ✨ feat: sendPoll quote msg. single select by default #3045 [`46b432f`](https://github.com/open-wa/wa-automate-nodejs/commit/46b432f12fbb112e206556d53e9c7b95ca432b37)
- updated types-only package [`6dbfcdf`](https://github.com/open-wa/wa-automate-nodejs/commit/6dbfcdf8c12c0cc51571c878e41946a89c7ade7a)
- 💚 fix ci build for sendPoll #3045 [`34677c4`](https://github.com/open-wa/wa-automate-nodejs/commit/34677c4216a9df7b6ce281c9e4e379e2ee0d3f05)
- 🥚 patches update [`229ba87`](https://github.com/open-wa/wa-automate-nodejs/commit/229ba87ceec261d0420c0a071cb86065882399f5)
- 🥚 patches update [`7b7483d`](https://github.com/open-wa/wa-automate-nodejs/commit/7b7483d2ba44bf24c269ce6cedab51a77973d603)
- 🥚 patches update [`49f6fbd`](https://github.com/open-wa/wa-automate-nodejs/commit/49f6fbdd4407115aaf3845392936fcb207d7c058)
- 🥚 patches update [`ee3a7a0`](https://github.com/open-wa/wa-automate-nodejs/commit/ee3a7a0b6716707f3c2dced200fba622f4c78c73)
- 🥚 patches update [`ad147c0`](https://github.com/open-wa/wa-automate-nodejs/commit/ad147c0d292d12cda08ff56f005a3530e7bd8967)
- 🥚 patches update [`bf42ba7`](https://github.com/open-wa/wa-automate-nodejs/commit/bf42ba715dd4530544badb0bf83b97a3b12f5e65)
- updated types-only package [`fccfd3a`](https://github.com/open-wa/wa-automate-nodejs/commit/fccfd3a3ee1e4494bfa2bfbf40babe524bc955ee)
